### PR TITLE
Add handled exception properties

### PR DESCRIPTION
**Pull request by guperrot**

### CODE REVIEW

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.UWP/Package.appxmanifest [hunk 1]
```diff
@@ -24,5 +24,6 @@
   </Applications>
   <Capabilities>
     <Capability Name="internetClient" />
+    <Capability Name="privateNetworkClientServer" />
```
Comment (guperrot) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.UWP/Package.appxmanifest]: For Rum to work reliably, could be in another PR but hey

File: SDK/AppCenter/Microsoft.AppCenter.Android.Bindings/Microsoft.AppCenter.Android.Bindings.csproj [hunk 2]
```diff
@@ -16,6 +16,15 @@
...
     </AndroidTlsProvider>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
```
Comment (guperrot) [inline SDK/AppCenter/Microsoft.AppCenter.Android.Bindings/Microsoft.AppCenter.Android.Bindings.csproj]: Visual Studio for Mac will insist on copying the root commands to each configuration everytime we build, so removed at root to avoid having 3 times the commands instead of 2...

File: SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Android/Microsoft.AppCenter.Crashes.Android.csproj [hunk 3]
```diff
@@ -48,14 +48,12 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="PlatformCrashes.cs" />
+    <Compile Include="Crashes.cs" />
```
Comment (guperrot) [inline SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Android/Microsoft.AppCenter.Crashes.Android.csproj]: Refactoring to using the static partial pattern like in core/push/distribute (analytics still need to align too).

File: SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/TestCrashException.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
```
Comment (guperrot) [inline SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/TestCrashException.cs]: UWP Crashes.GenerateTestCrash will now crash.

File: build.cake [hunk 5]
```diff
@@ -104,7 +104,7 @@ Task("Externals-Android")
     var pushLib = "appcenter-push-release";
     var pushLibFile = $"{AndroidExternals}/{pushLib}.aar";
     var pushLibUnzippedPath = $"{AndroidExternals}/{pushLib}";
-    var manifestUpdateFile = "SDK/AppCenterPush/Microsoft.AppCenter.Push.Android.Bindings/Properties/AndroidManifest.xml";
+    var manifestUpdateFile = "SDK/AppCenterPush/Microsoft.AppCenter.Push.Android.Bindings/AndroidManifest.xml";
```
Comment (guperrot) [inline build.cake]: This was causing new version of Visual Studio for Mac to believe the library project was an application and was updating the csproj...

File: scripts/configuration/ac-build-config.xml [hunk 6]
```diff
@@ -3,7 +3,7 @@
     <sdkInfo>
         <sdkVersion>1.1.1-SNAPSHOT</sdkVersion>
         <iosVersion>1.1.0</iosVersion>
-        <androidVersion>1.1.0</androidVersion>
+        <androidVersion>1.1.1-15+e7cb117b</androidVersion>
```
Comment (guperrot) [inline scripts/configuration/ac-build-config.xml]: The dont-merge-yet is only because of this

File: SDK/AppCenterPush/Microsoft.AppCenter.Push.Android.Bindings/Microsoft.AppCenter.Push.Android.Bindings.csproj [hunk 7]
```diff
@@ -24,7 +24,6 @@
     <WarningLevel>4</WarningLevel>
     <AndroidLinkMode>None</AndroidLinkMode>
     <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
-    <AndroidClassParser>class-parse</AndroidClassParser>
```
Comment (guperrot) [inline SDK/AppCenterPush/Microsoft.AppCenter.Push.Android.Bindings/Microsoft.AppCenter.Push.Android.Bindings.csproj]: new visual studio for mac is updating csproj automatically, I checked UI the value does not change, seems like the new default value

File: SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs [hunk 8]
```diff
@@ -11,20 +14,16 @@ public partial class Crashes : IAppCenterService
     {
         public string ServiceName => "Crashes";
 
+        /// <inheritdoc />
```
Comment (achocron) [inline SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs]: Does this actually work for you? I was never able to get it working for me

Comment (guperrot) [reply SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs]: It does not do anything visible except removing the warning

**Comment by dhei** [comment]
@TroubleMakerBen @guperrot there's a merge conflict in ac-build.config.xml

**Comment by guperrot** [comment]
Conflict resolved @dhei
